bazel/third_party/openssl/BUILD.openssl.bazel

```diff
@@ -0,0 +1,149 @@
+# Copyright Materialize, Inc. and contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License in the LICENSE file at the
+# root of this repository, or online at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+A build file for OpenSSL that is compatible with the Rust crate openssl-sys.
```

Comment:
How is this file to be updated when our version of openssl-sys changes and it updated its build.rs? Do we have any way to detect or flag that? Is this a thing we will only find later on?

Maybe a better question is: will bazel ever be used for production deployments? If not, then this might not matter as much and we can fix fallout if we find it. If we do expect to use bazel in prod then seems like we should have something automated to detect and flag build.rs changes for crates for which we have custom bazel builds.

Reply:
At the moment we'll have to manually update it. We should be able to automatically lint for this by using Bazel's own build graph to make sure the two stay in sync!

> will bazel ever be used for production deployments?

Yes I believe so. Mostly because IMO the binary that we run testdrive and other test suites against should be the actual binary we deploy to customer environments